.env.example

```diff
@@ -73,5 +73,7 @@ ENABLE_WW1_FULLTEXT_SEARCH=0
 DISQUS_SHORTNAME=shortname-here
 
 # Enable authentication for User galleries (XX), enabled by default
-# portal must be running on https (port 3001 for localhost)
+# portal must be running on https 
+# for localhost use https://127.0.0.1:3000 (dev) or https://127.0.0.1:3001 (docker), public/private pair of keys must be present in the app root dir
 ENABLE_XX_USER_AUTH=1
+OAUTH_URL=https://auth.europeana.eu/auth/realms/europeana/protocol/openid-connect
```

Comment:
Could you add a newline at the end of the file?

Reply:
done